### DISCUSSION TITLE
Extension doesn't work on firefox

### DISCUSSION THREAD

**Discussion by f-x1-1** [subhra74/xdm#969] [Q&A]
As you can see in the screenshots below the extension disables itself, I have tried old and new one, also tried to manually register extension ID still doesn't work.

![firefox_goe9CYA2Eh](https://user-images.githubusercontent.com/77340515/210250228-991edc73-12db-4161-8fd5-a8040d570a56.png)
![zrrA5R2gxe](https://user-images.githubusercontent.com/77340515/210250232-27a8337f-e487-4768-ae6e-689fac7ca287.png)
![xdm-app_I6y8J5BVnz](https://user-images.githubusercontent.com/77340515/210250234-5bb3efde-7fe7-4cb5-97c7-5b7d8a5185fa.png)

**Comment by subhra74** [comment]
Please try latest version 8.0.18, and let me know if this solves the issue. Please note after installing the new version you need to go to browser integration page and click on appropriate browser integration button to fix the browser integration issues
https://github.com/subhra74/xdm-experimental-binaries/releases/tag/8.0.18-beta
